R/dashbioAlignmentChart.R

```diff
@@ -0,0 +1,16 @@
+# AUTO GENERATED FILE - DO NOT EDIT
+
+dashbioAlignmentChart <- function(id=NULL, eventDatum=NULL, data=NULL, extension=NULL, colorscale=NULL, opacity=NULL, textcolor=NULL, textsize=NULL, showlabel=NULL, showid=NULL, showconservation=NULL, conservationcolor=NULL, conservationcolorscale=NULL, conservationopacity=NULL, conservationmethod=NULL, correctgap=NULL, showgap=NULL, gapcolor=NULL, gapcolorscale=NULL, gapopacity=NULL, groupbars=NULL, showconsensus=NULL, tilewidth=NULL, tileheight=NULL, overview=NULL, numtiles=NULL, scrollskip=NULL, tickstart=NULL, ticksteps=NULL, width=NULL, height=NULL) {
+    
+    component <- list(
+        props = list(id=id, eventDatum=eventDatum, data=data, extension=extension, colorscale=colorscale, opacity=opacity, textcolor=textcolor, textsize=textsize, showlabel=showlabel, showid=showid, showconservation=showconservation, conservationcolor=conservationcolor, conservationcolorscale=conservationcolorscale, conservationopacity=conservationopacity, conservationmethod=conservationmethod, correctgap=correctgap, showgap=showgap, gapcolor=gapcolor, gapcolorscale=gapcolorscale, gapopacity=gapopacity, groupbars=groupbars, showconsensus=showconsensus, tilewidth=tilewidth, tileheight=tileheight, overview=overview, numtiles=numtiles, scrollskip=scrollskip, tickstart=tickstart, ticksteps=ticksteps, width=width, height=height),
+        type = 'AlignmentChart',
+        namespace = 'dash_bio',
+        propNames = c('id', 'eventDatum', 'data', 'extension', 'colorscale', 'opacity', 'textcolor', 'textsize', 'showlabel', 'showid', 'showconservation', 'conservationcolor', 'conservationcolorscale', 'conservationopacity', 'conservationmethod', 'correctgap', 'showgap', 'gapcolor', 'gapcolorscale', 'gapopacity', 'groupbars', 'showconsensus', 'tilewidth', 'tileheight', 'overview', 'numtiles', 'scrollskip', 'tickstart', 'ticksteps', 'width', 'height'),
+        package = 'dashBio'
+        )
+
+    component$props <- filter_null(component$props)
+
+    structure(component, class = c('dash_component', 'list'))
+}
```

Comment:
Could you ensure all files end with a newline? I would feel bad asking you to edit manually auto-generated files, but I see that other auto-generated files in this PR have a newline character at EOF...

Reply:
@mkcor Sure; the reason these don't have them is that they were autogenerated in Python, and the original 'transpiler' file template doesn't have a newline at EOF. 

Easy enough to add, since I'm working on `dash-generate-components` this week.

Reply:
fixed in [`440d23b565d3845f163e85704aaa4559cd90013f`](https://github.com/plotly/dash/commit/440d23b565d3845f163e85704aaa4559cd90013f) while addressing https://github.com/plotly/dash/issues/711

Reply:
Great! So, all files will automatically have a newline at EOF next time the package is updated -- most likely not next time, actually, since we'll need PR https://github.com/plotly/dash/pull/725 merged in *and* dash-bio's migration to React 16 + Dash 0.42 ...

Do you mind fixing the missing newlines in this PR?

Reply:
Not at all! I just regenerated all the functions and their documentation pages with newlines at EOF. All future package builds using `dash-generate-components` related to all other Dash component libraries will have newlines at EOF as well.

fixed in f95e14e